src/components/InputMoney.js

```diff
@@ -11,11 +11,12 @@ const SContainer = styled.div`
 
 const SPrefix = styled.div`
   font-size: 16px;
-  padding-top: 19px;
+  padding-top: 21px;
   padding-right: 8px;
   color: ${props => (props.active ? "#545454" : "#c6c6c6")};
   opacity: ${props => (props.focus ? 1 : 0)};
   position: absolute;
+  font-family: 'Roboto', sans-serif;
```

Comment:
You put this in i`<InputMoney />` but `<InputAmountProgress />`does not extend this component so you need to apply it there